mesonbuild/compilers/d.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os.path, subprocess
+import os.path, re, subprocess
```

Comment:
We've been doing an inconsistent job of this, but I think we're moving to top level imports as one per line but multiple import in a `from module import ...`, i.e.:
```python
import re
import typing

from foo import a, b, c
```